src/styl/_themes/_skin.styl

```
@@ -33,7 +33,7 @@
     &-chat
     &-exclusive
     &-fakeoff
-    &-lastminute
+    &-alerteinfo
```

It should be in english (and camelcase since colette v4)
```suggestion
    &-alertInfo
```